### ISSUE TITLE
Sharing fails in 6.4.0

### ISSUE THREAD

**Issue by d7oc** [owncloud/ocis#10041]
## Describe the bug
If the share dialog in oCIS is opened you only see an animation, but nothing happens:

https://github.com/user-attachments/assets/b425a941-0264-4dd7-82da-f2a7575c61c2

The permissions call is failing with a 500:

<img width="1154" alt="Screenshot 2024-09-12 at 16 14 39" src="https://github.com/user-attachments/assets/b7b61f2d-9685-406e-b284-274dbc70b134">

Proxy logs shows:

```json
{"level":"debug","service":"proxy","claims":"marshaling error: json: unsupported type: map[interface {}]interface {}","user":{"id":{"idp":"https://ocis.kube.owncloud.test","opaque_id":"024b8111-1c82-4d29-91ad-428dd60b6f69","type":1},"username":"admin","
mail":"admin@example.org","display_name":"Admin","groups":["509a9dcd-bb37-4f4f-a01a-19dca27d9cfa"],"opaque":{"map":{"roles":{"decoder":"json","value":"WyI3MTg4MTg4My0xNzY4LTQ2YmQtYTI0ZC1hMzU2YTJhZmRmN2YiXQ=="}}},"uid_number":99,"gid_number":99},"time":"
2024-09-12T14:04:50Z","line":"github.com/owncloud/ocis/v2/services/proxy/pkg/middleware/account_resolver.go:179","message":"associated claims with user"}
{"level":"debug","service":"proxy","claims":"marshaling error: json: unsupported type: map[interface {}]interface {}","time":"2024-09-12T14:04:51Z","line":"github.com/owncloud/ocis/v2/services/proxy/pkg/middleware/oidc_auth.go:70","message":"cache hit f
or userinfo"}
{"level":"debug","service":"proxy","claims":"marshaling error: json: unsupported type: map[interface {}]interface {}","user":{"id":{"idp":"https://ocis.kube.owncloud.test","opaque_id":"024b8111-1c82-4d29-91ad-428dd60b6f69","type":1},"username":"admin","
mail":"admin@example.org","display_name":"Admin","groups":["509a9dcd-bb37-4f4f-a01a-19dca27d9cfa"],"opaque":{"map":{"roles":{"decoder":"json","value":"WyI3MTg4MTg4My0xNzY4LTQ2YmQtYTI0ZC1hMzU2YTJhZmRmN2YiXQ=="}}},"uid_number":99,"gid_number":99},"time":"
2024-09-12T14:04:51Z","line":"github.com/owncloud/ocis/v2/services/proxy/pkg/middleware/account_resolver.go:179","message":"associated claims with user"}
{"level":"debug","service":"proxy","claims":"marshaling error: json: unsupported type: map[interface {}]interface {}","time":"2024-09-12T14:04:57Z","line":"github.com/owncloud/ocis/v2/services/proxy/pkg/middleware/oidc_auth.go:70","message":"cache hit f
or userinfo"}
{"level":"debug","service":"proxy","claims":"marshaling error: json: unsupported type: map[interface {}]interface {}","time":"2024-09-12T14:04:57Z","line":"github.com/owncloud/ocis/v2/services/proxy/pkg/middleware/oidc_auth.go:70","message":"cache hit f
or userinfo"}
{"level":"debug","service":"proxy","claims":"marshaling error: json: unsupported type: map[interface {}]interface {}","user":{"id":{"idp":"https://ocis.kube.owncloud.test","opaque_id":"024b8111-1c82-4d29-91ad-428dd60b6f69","type":1},"username":"admin","
mail":"admin@example.org","display_name":"Admin","groups":["509a9dcd-bb37-4f4f-a01a-19dca27d9cfa"],"opaque":{"map":{"roles":{"decoder":"json","value":"WyI3MTg4MTg4My0xNzY4LTQ2YmQtYTI0ZC1hMzU2YTJhZmRmN2YiXQ=="}}},"uid_number":99,"gid_number":99},"time":"
2024-09-12T14:04:57Z","line":"github.com/owncloud/ocis/v2/services/proxy/pkg/middleware/account_resolver.go:179","message":"associated claims with user"}
{"level":"debug","service":"proxy","claims":"marshaling error: json: unsupported type: map[interface {}]interface {}","user":{"id":{"idp":"https://ocis.kube.owncloud.test","opaque_id":"024b8111-1c82-4d29-91ad-428dd60b6f69","type":1},"username":"admin","
mail":"admin@example.org","display_name":"Admin","groups":["509a9dcd-bb37-4f4f-a01a-19dca27d9cfa"],"opaque":{"map":{"roles":{"decoder":"json","value":"WyI3MTg4MTg4My0xNzY4LTQ2YmQtYTI0ZC1hMzU2YTJhZmRmN2YiXQ=="}}},"uid_number":99,"gid_number":99},"time":"
2024-09-12T14:04:57Z","line":"github.com/owncloud/ocis/v2/services/proxy/pkg/middleware/account_resolver.go:179","message":"associated claims with user"}
```

## Steps to reproduce
1. Login
2. Click on the share button of a folder

## Expected behavior
Share dialog should open

## Actual behavior
Share dialog does not open correctly

## Setup
Use k9s setup from https://github.com/owncloud/ocis-charts/pull/701. Just execute `helmfile sync` in `deployments/ocm-install`

## Additional context
System was updated from 6.3.0

**Comment by micbar** [comment]
How can that slip through our tests? We do ugrade tests and a ton of other api tests on that endpoint.

**Comment by wkloucek** [comment]
I couldn't reproduce this during quick QA in https://github.com/owncloud/ocis-charts/pull/747

**Comment by wkloucek** [comment]
Update from my side: I didn't manage to share again with oCIS 6.4.0. Don't know what I did before

**Comment by butonic** [comment]
fixed by https://github.com/owncloud/ocis/pull/10050 the single binary picks up the jjwt secret because other services configure it and the sharedConfig in reva is a package level singleton.

**Comment by saw-jan** [comment]
Is it a config issue? we can think of a possible test scenario if it is reproduceable using single binary setup.

**Comment by micbar** [comment]
This config fix has been released with ocis 6.4.0
